components/tag-connect/common.stanza

```diff
@@ -0,0 +1,60 @@
+#use-added-syntax(jitx)
+defpackage ocdb/tag-connect/common : 
```

Comment:
This pin usage is common to arm 10 pin headers, it may be better in utils under "debug-modules" or something.